pubspec.yaml

```diff
@@ -18,14 +18,14 @@ publish_to: 'none' # Remove this line if you wish to publish to pub.dev
 version: 1.0.0+1
 
 environment:
-  sdk: ">=2.7.0 <3.0.0"
+  sdk: ">=2.17.5 <3.0.0"
+  flutter: ">=3.0.0"
 
 dependencies:
+  auth0_flutter: ^1.0.2
   flutter:
     sdk: flutter
-  http: ^0.12.1
-  flutter_appauth: ^0.9.1
-  flutter_secure_storage: ^3.3.3
+  http: ^0.13.5
```

Comment:
Is this library still in use?

Reply:
I missed to clean that one up, thanks for catching that!